modules/tinymce/src/models/dom/main/ts/table/api/TableCellSelection.ts

```diff
@@ -95,6 +92,7 @@ export const TableCellSelection = (editor: Editor): TableCellSelection => {
 
     const keydown = (event: KeyboardEvent) => {
       const wrappedEvent = DomEvent.fromRawEvent(event);
+      // TODO: Using command causes issue with undo state
       editor.execCommand('TableResizeHandlerHide', false, undefined, { skip_focus: true });
```

Comment:
From what I can see this is only used internally inside the model do we need this to be a command? I think none of the commands are really required since it's about modules internally taking to each other within the model. I don't think we need to expose these at an api level.